### ISSUE TITLE
AttributeError: 'str' object has no attribute 'decode' when trying to run simulation server with GPU

### ISSUE THREAD

**Issue by sardine8042** [cyberbotics/webots-server#22]
Hey guys, great job on the repo so far. Here's a simple error I encountered when trying to start the servers with a GPU:

```
deployments-webots-1  | Traceback (most recent call last):
deployments-webots-1  |   File "/usr/local/lib/python3.8/dist-packages/tornado/web.py", line 1711, in _execute
deployments-webots-1  |     result = method(*self.path_args, **self.path_kwargs)
deployments-webots-1  |   File "simulation_server.py", line 633, in get
deployments-webots-1  |     gpu = nvmlDeviceGetName(nvmlHandle).decode('utf-8')
deployments-webots-1  | AttributeError: 'str' object has no attribute 'decode'
```

The fix is simple -- I just removed the call to `decode`, since `nvmlDeviceGetName` returns a `str` already.

Not sure if this is reproducible on other platforms: I'm running this in a Docker container on Windows 10 (x86_64) with GPU passthrough enabled via Docker Desktop on the WSL 2 engine.

If you want to make line `simulation_server.py:633` more resilient, I'd suggest adding a check for a byte or str before attempting to decode.

Thanks!

**Comment by omichel** [comment]
This problem was probably caused by a very recent change (5 days ago) in the pynvml module:
https://github.com/gpuopenanalytics/pynvml/pull/46/files#diff-c9c6160b3107b52ed919117b3cba3bd81f8539baa056cf1e8693329a26ba5bc7R2009

**Comment by BenjaminDeleze** [comment]
Thank you!
I fixed it in #23
